### PR TITLE
Add Pictures with Links to Contacts Page

### PR DESCRIPTION
**Pull request by elijahhiggso**
Added pictures with links to each persons Linkedin, and also formatted the contacts page.

Before:
image

After:
image

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -116,3 +116,17 @@ h5 {
   line-height: .1;
   font-weight: bold;
 }
+*{
```
Comment (jojojames) [inline src/main/webapp/style.css:119]: What is this?

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -116,3 +116,17 @@ h5 {
...
+  box-sizing: border-box;
+}
+.column {
+  float: left;
+  width: 33.33%;
+  padding: 5px;
+}
+/* Clearfix (clear floats) */
```
Comment (jojojames) [inline src/main/webapp/style.css:127]: What does this mean?

Comment (elijahhiggso) [reply src/main/webapp/style.css:127]: I used a clearfix in order to line up the pictures side by side in the while inside of the card.

File: src/main/webapp/style.css (hunk 1 above)
Comment (elijahhiggso) [reply src/main/webapp/style.css:119]: The * selects all the elements to make it look the same regardless

**Pull request merged**